doc/classes/CodeEdit.xml

```diff
@@ -604,6 +604,12 @@
 				Emitted when the user requests code completion. This signal will not be sent if [method _request_code_completion] is overridden or [member code_completion_enabled] is [code]false[/code].
 			</description>
 		</signal>
+		<signal name="refactor_requested">
+			<param index="0" name="type" type="int" />
+			<description>
+				Emitted when the user requests a refactor.
+			</description>
+		</signal>
```

Comment:
Oh my god, this is particularly egregious. In its current state (prone to future changes), it should not be exposed at all, let alone without its corresponding enum. 

CC @KoBeWi because of https://github.com/godotengine/godot/pull/69012#discussion_r2051790554. We somehow still can't choose to keep signals unexposed.

Reply:
CodeEdit has `code_completion_requested` too, so it's not completely out of place.

...if not the fact that this signal is only emitted externally.

Reply:
It looks like signals can be hidden from docs as of recently.
- https://github.com/godotengine/godot/pull/105691

Reply:
I'm not sure that means they're completely hidden from the API, though. Things like autocompletion may recommend them, the Node dock may display them. GDExtension may have them exposed anyway, though the leading underscore is a deterrent.

Reply:
My only concern with this PR is still this specific signal. @adamscott , can you consider a workaround as KoBeWi also discussed in https://github.com/godotengine/godot/pull/69012#discussion_r2051790554?
It would be an admittedly rare, but not unique case to use `Object.add_user_signal()` for this purpose.
https://github.com/godotengine/godot/blob/7864ac80192e9e91bf56176af9f04cc013b580aa/scene/2d/sprite_2d.cpp#L556
https://github.com/godotengine/godot/blob/7864ac80192e9e91bf56176af9f04cc013b580aa/scene/gui/code_edit.cpp#L3906

Reply:
It could also just be removed, like I suggest here https://github.com/godotengine/godot/pull/102380#discussion_r2099243262
There's no need for it in CodeEdit.